permits/admin.py

```diff
@@ -52,10 +50,6 @@
     "delete_group",
     "see_private_requests",
     # DRF Token authentication
-    "view_tokenproxy",
-    "add_tokenproxy",
-    "change_tokenproxy",
-    "delete_tokenproxy",
```

Comment:
I did not understand what exactly this was good for, I guess we can't just delete it.